package.json

```
@@ -38,7 +38,7 @@
     "dependencies": {
         "resolve": "^1.15.1",
         "source-map": "^0.7.3",
-        "typescript": "^3.9.2"
+        "typescript": "^4.0.2"
```

`^` doesn't make much sense with typescript's versioning scheme anyway
```suggestion
        "typescript": ">=4.0.2"
```